e2e/src/transaction-exclusion.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { describe, expect, it } from "@jest/globals";
+import { config } from "./config/tests-config";
+import { etherToWei, getTransactionHash, getWallet, TransactionExclusionClient, wait } from "./common/utils";
+import { TransactionRequest } from "ethers";
+
+const l2AccountManager = config.getL2AccountManager();
+
+describe("Transaction exclusion test suite", () => {
+  it.concurrent(
+    "Should get the status of the rejected transaction reported from Besu RPC node",
+    async () => {
+      if (!config.getTransactionExclusionEndpoint()) {
+        // Skip this test if transaction exclusion endpoint is not defined
+        return;
```

Comment:
this is bad practice, Tests shall not be short-circuit like this! The report will say the test passed and it did not even if the endpoint is undefined.

So, either:
a) when no endpoint is defined we report as skipped;
b) we make the endpoint mandatory and the test fails if not defined;

Reply:
I will add an assertion to check if config.getTransactionExclusionEndpoint() is defined